src/ports/postgres/modules/utilities/pivot.py_in

```diff
@@ -222,59 +261,78 @@ def pivot(schema_madlib, source_table, out_table, index, pivot_cols,
 
                         # Collect pcol values for the dict
                         if output_col_dictionary:
-                            insert_str.append("{0}".format(
+                            dict_insert_str.append("'{0}'".format(
```

Comment:
Is there a particular reason this is needed? The install-check fails on NULL values and removing them seems to solve the issue.